public/app/plugins/datasource/prometheus/partials/annotations.editor.html

```diff
@@ -27,4 +27,13 @@ <h5 class="section-heading">Field formats</h6>
 			</div>
 		</div>
 	</div>
+
+	<h5 class="section-heading">Other options</h5>
+	<div class="gf-form-inline">
+		<div class="gf-form">
+			<gf-form-switch class="gf-form" label="Series value as timestamp" label-class="width-14" checked="ctrl.annotation.useValueForTime"
+			 tooltip="The unit of timestamp is milliseconds. If unit of series value is seconds, multiply 1000.">
```

Comment:
"If the unit of the series value is seconds, multiply its range vector by 1000."